appveyor.yml

```diff
@@ -9,15 +9,15 @@ install:
   # Get the latest stable version of Node.js
   - ps: Install-Product node $env:nodejs_version
   # install modules
-  - npm ci
+  - yarn
 
 # Post-install test scripts.
 test_script:
   # Output useful info for debugging.
   - node --version
-  - npm --version
+  - yarn --version
   # run tests
-  - npm run test-win
+  - yarn test
```

Comment:
Is this supposed to be `test-win`?

Reply:
Back in da day we used to not be able to spawn windows commands haha. It was rough